thermal_processing.py

```diff
@@ -80,15 +84,12 @@ def classify(recording, api, s3):
     logging.info("uploading %s", video_filename)
     new_key = s3.upload(video_filename)
 
-    # delete track positions before saving metadata
-    for track in formatted_tracks:
-        del track["positions"]
-
-    metadata = {"additionalMetadata": {"tracks": formatted_tracks}}
+    metadata = {"additionalMetadata": {
+        "algorithm" : algorithm_id,
```

Comment:
why is this going into additionalMetadata when we also have it against the track?

Reply:
Because:
1. sometimes there aren't any tracks.  (which could be a problem if there is a missed animal)
2. there are also still some tags - eg multiple animals that end up on the recording.